### ISSUE TITLE
DataTable custom filter not working when using scrollable attribute

### ISSUE THREAD

**Issue by asnp87** [primefaces/primeng#1763]
**I'm submitting a ...**  (check one with "x")
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**
http://plnkr.co/edit/Ccb7Op

**Current behavior**
Custom filter isn't showed correctly when using a scrollable datatable.

**Expected behavior**
The custom filter is showed correctly.

* **Angular version:** 2.0.X
2.4.2

* **PrimeNG version:** 2.0.X
1.1.4

**Comment by siddharthaReddy** [comment]
I am facing a similar issue. When scrollable is set to true, following bugs appear:

1. Custom filter on one of the column (dropdown) doesn't work. 
2. Header column for multiple selection checkbox disappears.
3. Sorting acts weird and clicking up on a column messes up the whole table and cells.

**Comment by DeeVeX** [comment]
my personal fix is to write this rule:
`div.ui-datatable-scrollable-header {
    overflow: visible;
}`
May break other features, but works for me like this.

for those who want to take a closer look:
primeng/components/datatable/datatable.css

**Comment by asnp87** [comment]
Fixed since 2.0.0

**Comment by prabharj** [comment]
I'm also facing the same issue. I'm using PrimeNG 2.0.4 with Angular 2. The suggestion given by @DeeVeX  worked for me.
